packages/ffe-skeleton-react/src/SkeletonCircle.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import classNames from 'classnames';
+
+export interface SkeletonCircleProps extends React.HTMLAttributes<HTMLDivElement> {
+    size?: 'sm' | 'md' | 'lg' | 'xl';
+}
+
+export const SkeletonCircle: React.FC<SkeletonCircleProps> = ({
+    size = 'lg',
+    className,
+    ...props
+}) => {
+    return (
+        <div
+            className={classNames(
+                'ffe-skeleton ffe-skeleton__circle',
+                `ffe-skeleton__circle--${size}`,
+                className,
+            )}
+            aria-hidden={true}
```

Comment:
Litt usikker på om `aria-hidden` egentlig er nødvendig her, men når jeg prøvde å søke det opp fikk jeg at det var anbefalt for at ikke skjermlesere skal pause? Men mulig jeg misforstod og det kun var når man brukte aria-busy